draft-ietf-quic-tls.md

```diff
@@ -1821,6 +1821,9 @@ QUIC (such as the use of TLS with TCP defined in {{!TLS13}}).  A fatal
 unsupported_extension alert MUST be sent by an implementation that supports this
 extension if the extension is received when the transport is not QUIC.
 
+Negotiating the quic_transport_parameters extension causes the EndOfEarlyData to
```

Comment:
```suggestion
Negotiating the quic_transport_parameters extension causes the requirement to send EndOfEarlyData to
```

Reply:
I think that the shorter phrasing is better.  It's not about not having a requirement to send EOED, but a requirement to not send EOED.  And, as this refers to the very next section, the fewer words expended here the better.

Reply:
For the question is whether an implementer (bot being on mind you) would be confused by the lack of "requirement to send" phrase, I tend to think that without it's still clear that you don't send EOED.